exercises/1-the-manual-menace/README.md

```diff
@@ -149,6 +149,8 @@ test:
 8. In the `inventory/host_vars/projects-and-policies.yml` file; add the new objects for the projects you want to create (dev & test) by adding another object to the content array for each. You can copy and paste them from the `ci-cd` example and update them accordingly. If you do this; remember to change the params_from_vars variable! e.g.
 
 ```yaml
+   # inventory/host_vars/projects-and-policies.yml
```

Comment:
I think it would be better to use something like `<kbd>` to call out the filename before the actual code block - i.e. for this case, it would be:

```
<kbd>*inventory/host_vars/projects-and-policies.yml*</kbd>
```yaml
 : 
 : 
```